### ISSUE TITLE
[CAIA Intake] Debt Resolution: VHA Payment History/One thing per page

### ISSUE THREAD

**Issue by mgleo** [department-of-veterans-affairs/va.gov-team#92519]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to request collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

## About your team

- **Team name:** `Debt Resolution`
- **OCTO product owner:** `Denise Coveyduc`
- **Product name:** `Debt Portal`
- **Product manager:** `Tom Davis`
- **Slack channel:** `[#debt-resolution](https://join.slack.com/share/enQtNzM5MTE3ODIwMDQ4My1hYTY3OGIyZGYzMGYwZjkyYWYyMzg5OTE4MDhiODRiOTU5ZGQ3ZGIzOTMyNTRjMzcwMjA4ZGZkZTE5MjNjY2Fm)`
- **Dedicated content writer on your team (if you have one):** `writer`
- **Dedicated a11y specialist on your team (if you have one):** `a11y`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [ ] Digitizing a new form
- [ ] Creating a new digital tool
- [x] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

Provide Veterans with timely, accurate and easy to digest information on payments made toward their copay balance. We will be receiving transaction data from a new source that will allow us to present this information on the Debt Portal faster and with improved readability.

Outcome summary:
- Improved clarity
- Provides confirmation and reassurance that their copay bill has been resolved after they have made a payment
- Reduced confusion and anger that Veterans feel (as seen in Medallia feedback)
- Reduced possibility of inaccurate information or outdated information (as seen in Medallia feedback)
- Improve findability and reduce cognitive burden by updating information architecture to follow the one thing per page design principal

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

- Initial discovery for new lighthouse data
- Mockups
- Research Plan draft

### Will you release this new product incrementally (for example, release to 25% of users to start)?

- [x]  Yes
- [ ]  No
 
 **Note:** For an incremental launch, we use React widgets to display dynamic content. We'll need at least 2 weeks of extra time to coordinate the content for this and work with your engineer. [Learn more about our process for dynamic content](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/sitewide-content/processes/dynamic-content-process.md). If your launch plans change, notify CAIA of the change as soon as possible so we can talk about how that will impact the timeline.

**If you are releasing incrementally:**
- **What dates do you anticipate starting and ending your incremental launch?** `TBD`
- **What is the name of your engineer who will build the React widget?** `TBD`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [x] Design intent 
- [ ] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide a link to your Collaboration Cycle Request:
- **[#91368](https://github.com/department-of-veterans-affairs/va.gov-team/issues/91368)** 

## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.

- [User flow](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=3107-43147&t=NnTloofgzU9q2cpZ-1)

- [Mockups (work in progress)](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=3212-30062&t=NnTloofgzU9q2cpZ-1)

## Next steps
- [ ] Review the remaining next steps in the **VFS team tasks** section here.
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well: [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work: [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team: [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next: [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

```[tasklist]
## Related Tickets
- [x] #91368
- [x] #93530
```

**Comment by kristinoletmuskat** [comment]
@josephrlee hello! I left a few comments in Figma on the VHA statement -- I'm not sure what is preexisting vs changes for this initiative, so feel free to backlog things that aren't part of the one-thing-per-page change. 

In the mocks, the final breadcrumb was missing from the statement -- make sure to add that in! We'd like to review the URLs for each of these levels too. Feel free to add to the figma and tag me or send them in the ticket, whatever makes the most sense. 

Are there areas you anticipate needing content support for this?

And noting the general feedback on the structure of the debt portal (that we discussed in slack yesterday, but just for visibility)-- we seem to be missing a way to see payments made towards the bill, as well as historic transaction info (both charges and payments) for debts that are paid off. While these might be edge cases (from the main use case of 'pay a bill'), we could be creating headache for veterans that need this information by putting some but not all of the info about their copays in this portal. From a taxonomy perspective we're missing this part of the experience for sure, but I think it's worth investigating the veteran mental model and needs re: historic payment and charge info.

Let me know if you have any Qs!

**Comment by mgleo** [comment]
Hi @kristinoletmuskat, Thanks for looking into this. I think you may be looking at a previous version of the VHA mockups. 

Here is the latest:
- [Mockups (work in progress)](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=3212-30062&t=NnTloofgzU9q2cpZ-1)
It is under the page _VHA Balance + One thing per page Exploration - WIP_ and the Artboard is titled _VHA - One thing per page_

The historic info is on the Copay detail page under _Previous Statements_. From there, users are able to view their past transactions on a separate page. This is an existing feature on staging. You can view the an example [page here](https://staging.va.gov/manage-va-debt/summary/copay-balances/c9b1b3ed-8515-4c49-81a9-4f63ae6b29e5/detail/statement) on staging under user 81. I don't believe we currently have an example where a staging user has paid their bill but it will be reflected on their statement. 
cc @josephrlee @charlottecesana

**Comment by kristinoletmuskat** [comment]
Hey @mgleo just confirming that this work is still open, it's just delayed a bit, right?

**Comment by mgleo** [comment]
Hi @kristinoletmuskat, that's correct, this work has not been completed yet.

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mnorthuis** [comment]
@mgleo Can you provide a status on this work.  Do you need any additional support form content and IA?

**Comment by mgleo** [comment]
Hi @mnorthuis, great timing! We were just discussing moving forward on the "One thing per page" aspect of this for both VHA and VBA. 

We won't be adding new info but breaking up the information and adding a Resolve page to both VBA and VHA sections on the Debt Portal. 

Below are before/after userflow docs and mockups.

- [Userflow](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=0-5771&t=pBbTb3vTUI8zIP7q-1)
- [VHA mock ups](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=4361-46550&t=pBbTb3vTUI8zIP7q-1)
- [VBA mock ups](https://www.figma.com/design/OiiDTTVTCf8j0GngRg0xxD/VA-Debt-Portal?node-id=4361-46551&t=pBbTb3vTUI8zIP7q-1)

cc @denisecoveyduc @Tom-Davis @HeatherRienks @charlottecesana

**Comment by HeatherRienks** [comment]
@mnorthuis - Hi! Is there any other feedback for us based on what Megan shared above? We are looking to move this forward and wanted to check in with you.

cc: @Tom-Davis @denisecoveyduc

**Comment by mnorthuis** [comment]
@HeatherRienks Will work on getting a content specialist and IA assigned out to this work.  Stay tuned!

**Comment by charlottecesana** [comment]
Hi @HeatherRienks  @mnorthuis! Thank you for re-sparking this conversation. Megan and I tested 1TPP in a study with Veterans last Fall, and the IA and content were validated (tested very well). I don't believe we need further assistance on this initiative - it is ready for development. It is currently in midpoint staging review and we are looking forward to seeing it go live to simplify the debt portal IA structure that tested so well in the study.

**Comment by VCM86** [comment]
> Hi [@HeatherRienks](https://github.com/HeatherRienks) [@mnorthuis](https://github.com/mnorthuis)! Thank you for re-sparking this conversation. Megan and I tested 1TPP in a study with Veterans last Fall, and the IA and content were validated (tested very well). I don't believe we need further assistance on this initiative - it is ready for development. It is currently in midpoint staging review and we are looking forward to seeing it go live to simplify the debt portal IA structure that tested so well in the study.

Thank you for this context, Charlotte! We'll go ahead and close out this ticket.
